css-view-transitions-2/Overview.bs

```diff
@@ -1080,8 +1081,24 @@ When capturing the old or new state for an element, perform the following steps
 
 ## Capturing and applying 'view-transition-group' ## {#vt-group-algorithm}
 
-<div algorithm="additional capture steps (group)">
-When capturing the old or new state for an element, perform the following steps given a [=captured element=] |capturedElement| and an [=element=] |element|:
+<div algorithm="additional old apture steps (group)">
+When capturing the old or new state for an element, perform the following steps given a [=captured element=] |capturedElement|, an [=element=] |element| and a {{ViewTransition}} |transition|:
+
+	1. Let |containingGroupName| be the [=used value=] of |element|'s ''view-transition-group''.
+
+	1. If |containingGroupName| is not null, then:
+
+		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][containingGroupName].
+
+		1. If |groupContainerElement| is not null, then:
+
+			1. Multiply |capturedElement|'s [=captured element/old transform=] by the inverse of |groupContainerElement|'s [=captured element/old transform=].
```

Comment:
"Set capture’s old transform to a <transform-function> that would map element’s border box from the group container element's border box to its current visual position."

This seems more explicit about the intent than the current statement.

Reply:
Done